pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -71,6 +74,14 @@ var podStateController controller.Controller
 const csiProvider = "fake.csi.provider"
 
 func TestScheduling(t *testing.T) {
+	fg := featuregate.NewFeatureGate()
```

Comment:
@tzneal I pushed this up (along w/ the static config to use 1.24.0) for visibility, but it doesn't appear to be a way to do this (I'm waiting to see the `should respect minDomains constraints` test case below fail)

Reply:
You'll need to add to https://github.com/aws/karpenter-core/blob/0c034ad78a21380fe2a43f5aeb93106dcf26b58d/.github/workflows/presubmit.yaml#L12 to get the 1.25 CI tests to run.  You can add 1.25 and 1.26, we should have added them both already.

Reply:
I don't think 1.25+ is a part of this story, as `MinDomainsInPodTopologySpread` was introduced in v1.24.0, right?